tc/graphene/python_worker/graphene_sgx/build_gsc_python_worker.sh

```diff
@@ -38,7 +38,7 @@ if [ "$GSC_IMAGE_EXISTS" = "yes" ]; then
 fi
 
 # Manifest files
-MANIFEST_FILE_DIR="${TCF_HOME}/examples/graphene_apps/python_worker/graphene/manifest"
+MANIFEST_FILE_DIR="${TCF_HOME}/tc/graphene/python_worker/graphene_sgx/manifest"
 MANIFEST_FILES="python.manifest
```

Comment:
python.manifest -> python_worker.manifest

Reply:
manifest file is reverted back to old one.